oauthenticator.py

```diff
@@ -21,10 +21,16 @@
 
 from traitlets import Unicode, Set
 
+# Support github.com and github enterprise installations
+GITHUB_HOST = os.environ.get('GITHUB_HOST', 'github.com')
```

Comment:
This should use `get('GITHUB_HOST') or 'github.com'` to handle the `GITHUB_HOST=''` case, which should behave the same as being unset.